Snowflake.Data.Tests/SFBindTestIT.cs

```diff
@@ -686,6 +687,83 @@ public void testPutArrayBind()
                 conn.Close();
             }
         }
+        
+        [Test]
+        public void testPutArrayBindWorkDespiteOfProblematicTypeNameHandlingValue()
+        {
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings {
+                TypeNameHandling = TypeNameHandling.Auto
+            };
+            
+            
+            using (IDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = ConnectionString;

```

Comment:
can we pass connection string in contrutor?

Reply:
yes